bin/client_build

```diff
@@ -13,7 +13,7 @@
 #> GOOS=windows GOARCH=amd64 bin/client_build
 #
 # Check if program has dynamic libraries:
-#> readelf -d /build/client/mysterium_client
+#> readelf -d build/client/mysterium_client
```

Comment:
There are no readelf on osx by default. We should add help notice how to get it.

Reply:
Fixed